modules/virt-creating-vm-quick-start-web.adoc

```diff
@@ -14,7 +14,6 @@ Quick Start tours for creating virtual machines include the following:
 
 * Creating a Red Hat Enterprise Linux virtual machine
 * Creating a Windows 10 virtual machine
```

Comment:
I think you can add "Customizing a boot source" if it is released. See [source](https://gitlab.cee.redhat.com/cpaas-midstream/openshift-virtualization/hyperconverged-cluster-operator/-/blob/cnv-4.10-rhel-8/extras/quickStart/customize_a_boot_source_quickstart.yaml.in).

Reply:
I'm thinking maybe we should generalize instead of attempting to list specific Quick Starts that will either be obsoleted, renamed, or otherwise changed. What about:

_Quick Start tours are available in the {product-title} web console to assist you in working with boot sources._